docs/guides/button-upgrade-guide.md

```diff
@@ -8,23 +8,29 @@ order: 4
-    - [Upgrade examples for variant="link" and has "padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-examples-for-link-variant-with-no-href-attribute-and-padding-overrides)
-    - [Upgrade examples for variant="link" and has "no padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-example-for-link-variant-with-no-href-attribute-and-no-padding-overrides)
-  - [Upgrading __CloseButton__](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton)
-    - [Upgrade examples for __CloseButton__](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton-upgrade-examples-for-closebutton)
+- [Upgrading **icon only** buttons, no visible text](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-icon,-icon-inverse,-circle-default,-circle-primary,-or-circle-danger)
 
+  - [Upgrade examples for "icon" or "icon-inverse"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-icon,-icon-inverse,-circle-default,-circle-primary,-or-circle-danger-upgrade-examples-for-icon-or-icon-inverse)
+  - [Upgrade examples for "circle-default", "circle-primary", or "circle-danger"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-icon,-icon-inverse,-circle-default,-circle-primary,-or-circle-danger-upgrade-examples-for-circle-default,-circle-primary,-or-circle-danger)
+
+- [Upgrading **link variant** values](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse)
+  - [Upgrade examples for variant=”link” and has "padding overrides plus href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-examples-for-link-variant-with-an-href-attribute-and-padding-overrides)
+  - [Upgrade examples for variant="link" and has "padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-examples-for-link-variant-with-no-href-attribute-and-padding-overrides)
+  - [Upgrade examples for variant="link" and has "no padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-example-for-link-variant-with-no-href-attribute-and-no-padding-overrides)
+- [Upgrading **CloseButton**](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton)
+  - [Upgrade examples for **CloseButton**](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton-upgrade-examples-for-closebutton)
+
+##### Note about Codemods
+
+These complex props are updated the `updateV7Props.js` codemod that is described in the v8 docs at <https://instructure.design/#v8-upgrade-guide>. These are in the V8 (and later) versions of the InstUI repo.
```

Comment:
```suggestion
These complex props are updated by the `updateV7Props.js` codemod that is described in the v8 docs at <https://instructure.design/#v8-upgrade-guide>. These are in the V8 (and later) versions of the InstUI repo.
```

Reply:
Just a note: when we release V9, we probably gonna have to go through the links in the V7 docs too, becasuse if we remove this guide from v9, it won't work anymore.

Reply:
yep, we'll need to do that